monai/engines/evaluator.py

```diff
@@ -142,7 +142,7 @@ def run(self, global_epoch: int = 1) -> None:  # type: ignore[override]
 
         """
         # init env value for current validation process
-        self.state.max_epochs = global_epoch
+        self.state.max_epochs = max(global_epoch, 1)  # at least one epoch of validation
         self.state.epoch = global_epoch - 1
```

Comment:
For validation at start, so here `self.state.max_epochs` will be 1, `self.state.epoch` will -1? Will it run 2 epochs?

Reply:
it'll not run for 2 epochs according to the stopping condition https://github.com/pytorch/ignite/blob/854b5ea91629aefc5ab6fc35b5990190e3b35178/ignite/engine/engine.py#L733-L741